6-scicat/5-data_catalog.md

```diff
@@ -17,8 +17,8 @@ The main components of SciCat are:
 - The backend, which provides all the data management functionalities and connects to the database.
 - The frontend, which is a single page web application providing a UI to most of the backend functionalities.
 
-More information about SciCat can be found on its [website](https://scicatproject.github.io/), and also the code repositories for the [frontend](https://github.com/SciCatProject/frontend) and [backend](https://github.com/SciCatProject/scicat-backend-next) repositories
+More information about SciCat can be found on its [website](http://www.scicatproject.org/), and also the code repositories for the [frontend](https://github.com/SciCatProject/frontend) and [backend](https://github.com/SciCatProject/scicat-backend-next) repositories
```

Comment:
Strange, it seems that it's `https` below, but `http` here?

Reply:
That is strange. I guess linkcheck found a different URL in those cases. https works, too. Good catch!